Resources/Prototypes/Entities/Objects/Weapons/Guns/Revolvers/revolvers.yml

```diff
@@ -56,6 +56,8 @@
     - state: mag-unshaded-3
       map: ["enum.GunVisualLayers.MagUnshaded"]
       shader: unshaded
+  - type: Gun
+    fireRate: 2.5
```

Comment:
This means the deckard will kill someone in a little over 1 second?

Reply:
> This means the deckard will kill someone in a little over 1 second?

i will change the speed to 2